### PR TITLE
Add test case for virsh domfsthaw

### PR DESCRIPTION
**Pull request by liuzzfnst**

### CODE REVIEW

**Comment by yangdongsheng** [comment]
There are two commits about "Merge", please drop them. Thanx for your patches

**Comment by waynesun09** [comment]
@liuzzfnst there is a guide mail for git and github usage, and you can take a look of it since your pr branches are not clean for review, checkout the link:

https://www.redhat.com/archives/autotest-kernel/2013-April/msg00076.html

**Comment by liuzzfnst** [comment]
@waynesun09  
Thank you very much.

File: libvirt/tests/src/virsh_cmd/domain/virsh_domfsthaw.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import logging
+import uuid
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domfsthaw.py]: Please remove unused import(s)

**Comment by PandaWei** [comment]
This is new testcases.
It's better to paste result here

File: libvirt/tests/src/virsh_cmd/domain/virsh_domfsthaw.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            if not vm.is_alive():
+                vm.start()
+        else:
+            vm.destroy()
+
+        if vm_ref == "none":
+            vm_name = " "
+
+        cmd_result = virsh.domfsthaw(vm_name, options=options, debug=True)
+
```
Comment (PandaWei) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domfsthaw.py]: why leave empty line here?

**Comment by liuzzfnst** [comment]
Test result on libvirt-1.2.8-16.el7.x86_64
<code>
# ./run -k --no-downloads -t libvirt --tests virsh.domfsthaw
SETUP: PASS (0.60 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/community/autotest/client/tests/virt-test/logs/run-2015-06-07-23.14.54/debug.log
TESTS: 8
(1/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.positive_tests.full_freeze: PASS (70.02 s)
(2/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.positive_tests.no_freezed_fs: PASS (48.89 s)
(3/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_qemu_guest_agent: PASS (36.60 s)
(4/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_start_qemu_guest_agent: PASS (53.91 s)
(5/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.not_active_domain: PASS (41.59 s)
(6/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_domain_ref: PASS (36.26 s)
(7/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.invalid_mountpoint: PASS (48.80 s)
(8/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.invalid_option: PASS (48.98 s)
TOTAL TIME: 385.11 s (06:25)
TESTS PASSED: 8
TESTS FAILED: 0
SUCCESS RATE: 100.00 %
</code>

**Comment by liuzzfnst** [comment]
Test result on libvirt-1.2.8-16.el7.x86_64
```
# ./run -k --no-downloads -t libvirt --tests virsh.domfsthaw
SETUP: PASS (0.60 s)
DATA DIR: /var/lib/virt_test
DEBUG LOG: /root/community/autotest/client/tests/virt-test/logs/run-2015-06-07-23.14.54/debug.log
TESTS: 8
(1/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.positive_tests.full_freeze: PASS (70.02 s)
(2/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.positive_tests.no_freezed_fs: PASS (48.89 s)
(3/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_qemu_guest_agent: PASS (36.60 s)
(4/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_start_qemu_guest_agent: PASS (53.91 s)
(5/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.not_active_domain: PASS (41.59 s)
(6/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.no_domain_ref: PASS (36.26 s)
(7/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.invalid_mountpoint: PASS (48.80 s)
(8/8) type_specific.io-github-autotest-libvirt.virsh.domfsthaw.negative_tests.invalid_option: PASS (48.98 s)
TOTAL TIME: 385.11 s (06:25)
TESTS PASSED: 8
TESTS FAILED: 0
SUCCESS RATE: 100.00 %
```

**Comment by PandaWei** [comment]
LGTM, 
ACK

**Comment by liuzzfnst** [comment]
Thanks.

**Comment by waynesun09** [comment]
ACK and merging now.
Thanks @liuzzfnst @PandaWei

**Pull request merged**